### ISSUE TITLE
.zshrc section clarification

### ISSUE THREAD

**Issue by MartinDelille** [tarjoilija/zgen#84]
I think this very nice plugin need some clarification about zshrc edition. If I understand well, if I modify my '.zshrc` file, I have to delete the `$ZGEN_INIT` file if I want my change to be effective at the next login right?

**Comment by Tuurlijk** [comment]
Yes, you should call `zgen reset` which removes ~/.zgen/init.zsh. Then the regeneration of this file will be triggered by the .zshrc on the next read of .zshrc.
